### DISCUSSION TITLE
Estimate is NA with Kemp

### DISCUSSION THREAD

**Discussion by JemmaSun** [adw96/breakaway#150] [Q&A]
Hi!
I would like to compare the richness of multiple samples collected in different years. I have multiple samples collected in each year, and would like to compare if there is any species richness changes in some of the years. Following the suggestion in the tutorial page, I used the kemp function to calculate the estimate richness for all my samples. However, some of my samples have no output estimations:

> Estimate of richness from method kemp:
  Estimate is NA
 with standard error NA

Just wondering what does 'NA' mean here? Can I use different model (i.e. the breakaway recommended model) for these samples and then group the estimates for comparisons? Thanks in advance!

**Comment by adw96** [comment]
Hi Jemma -- sometimes no Kemp models return non-negative estimates of the number of missing species. This is why we typically use Kemp estimators within `breakaway::breakaway`, which will choose the next most sensible estimator if Kemp models fail. Therefore, this issue should be resolved if you estimate with the function `breakaway` rather than `kemp`.
